kong/meta.lua

```diff
@@ -13,5 +13,14 @@ local version = setmetatable({
 return {
   _NAME = "kong",
   _VERSION = tostring(version),
-  _VERSION_TABLE = version
+  _VERSION_TABLE = version,
+
+  -- third-party dependencies' required version, as they would be specified
+  -- to lua-version's `set()`.
+  _DEPENDENCIES = {
+    nginx = "1.9.15.1",
+    --resty = "", -- not version dependent for now
+    serf  = "0.7.0",
+    --dnsmasq = "" -- not version dependent for now
```

Comment:
those need `from` and `to` fields, the function call in the previous places can have two arguments, where the second defaults to the first. If you specify them like this, it needs explicit to and from.

additionally; the resty version command `-V` also displays the nginx version it belongs to, so shouldn't we simply check for the same nginx version?